tests/kafkatest/services/console_consumer.py

```diff
@@ -114,13 +116,20 @@ def __init__(self, context, num_nodes, kafka, topic, message_validator=None, fro
         }
 
         self.consumer_timeout_ms = consumer_timeout_ms
+        for node in self.nodes:
+            node.version = version
 
         self.from_beginning = from_beginning
         self.message_validator = message_validator
         self.messages_consumed = {idx: [] for idx in range(1, num_nodes + 1)}
 
-    @property
-    def start_cmd(self):
+    def _kafka_dir(self, node):
```

Comment:
This doesn't seem like something that should be in this class -- the node is owned by this service, but is passed into the method. This seems more appropriate to be implemented once in the `KafkaService`.